stable/voyager/templates/deployment.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    chart: {{.Chart.Name}}-{{.Chart.Version}}
```

Comment:
Please use consistent formatting, i. e. space after `{{` and space before `}}`. Apply everywhere.